docs/content/concepts/assets/asset-auto-execution.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Auto-Materializing Assets | Dagster
+description: You can set up Dagster to automatically materialize assets when criteria are met.
+---
+
+# Auto-Materializing Assets <Experimental />
+
+You can set up Dagster to automatically materialize assets when criteria are met. This enables a declarative approach to asset scheduling – instead of defining imperative workflows to materialize your assets, you just describe the conditions under which they should be materialized.
```

Comment:
I think another benefit worth mentioning here is that auto-materialization makes it easier to add new things. You don't have to worry about building one mega dag, or linking different tags together via complex schedules. Dagster automatically figures all of this out and all you have to do is write the new asset.

Reply:
Is there wording you have in mind? "instead of defining imperative workflows" arguably captures this.

Reply:
In the interest of brevity I rescind this recommendation